__tests__/app.test.js

```diff
@@ -292,4 +292,22 @@ describe("PATCH /api/articles/:article_id", () => {
       expect(body.msg).toBe("Not Found")
     })
   })
+})
+
+//tests for deleting the comment 
+
+describe("DELETE the comment", () => {
+  test("deletes the comment with status 204", () => {
+    return request(app)
+    .delete('/api/comments/1')
+    .expect(204)
+  })
+  test("returns 404 if comment does not exist", () => {
+    return request(app)
+    .delete('/api/comments/999999')
+    .expect(404)
+    .then(({ body }) => {
+      expect(body.msg).toBe('Not Found');
+    });
```

Comment:
A good test suite so far✅
What if you tried to delete a comment at `/api/comments/not-an-id` ?